pandas/tests/arrays/sparse/test_array.py

```diff
@@ -696,14 +694,23 @@ def test_getslice_tuple(self):
         res = sparse[
             4:,
         ]  # noqa: E231
-        exp = SparseArray(dense[4:,])  # noqa: E231
+        exp = SparseArray(
+            dense[
+                4:,
+            ]
+        )  # noqa: E231
```

Comment:
Can the trailing comma be removed here? The explicit ignoring of `Missing whitespace after ',', ';', or ':' (E231)` suggests this is here on purpose

Reply:
Since `dense` is a 1D array, it should be the same to remove it I think